### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by kevmoo**
Branched from https://github.com/dart-lang/collection/pull/58

Will cleanup after that merges

### CODE REVIEW

**Comment by matanlurey** [comment]
Rebase?

**Comment by kevmoo** [comment]
Done, @matanlurey

File: lib/src/unmodifiable_wrappers.dart [hunk 1]
```diff
@@ -2,10 +2,10 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-export "dart:collection" show UnmodifiableListView, UnmodifiableMapView;
-
-import 'wrappers.dart';
 import 'empty_unmodifiable_set.dart';
+import 'wrappers.dart';
+
+export "dart:collection" show UnmodifiableListView, UnmodifiableMapView;
```
Comment (matanlurey) [inline lib/src/unmodifiable_wrappers.dart]: Consistent " versus ' while you are it.

File: test/algorithms_test.dart [hunk 2]
```diff
@@ -4,9 +4,10 @@
 
 /// Tests algorithm utilities.
 
+import 'dart:math';
```
Comment (matanlurey) [inline test/algorithms_test.dart]: Consistent " versus ' while you are it.

**Pull request closed without merging**